Aptoide/aptoide-client-v8
# [APP-959] terms and conditions dialog

**What does this PR do?**

This PR aims at implementing the changes to show the gdpr dialog everytime we open the app until we accepted it. 

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] MainPresenter.java

**How should this be manually tested?**

Open the app, check that you can only use it if you have accepted the gdpr. Check that on the next flows nothing shows up.
Confirm that everything is ok with the login as we removed the checkbox from that view.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-959](https://aptoide.atlassian.net/browse/APP-959)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
